test/Grains/TestFSharp/TestFSharp.fsproj

```diff
@@ -4,6 +4,11 @@
     <RootNamespace>TestFSharp</RootNamespace>
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
```

Comment:
We probably don't need this because there's already the change in Directory.Build.props.